Makefile

```diff
@@ -9,8 +9,9 @@ include tools/tools.mk
 
 build: out/vfkit
 
-test:
-	go test ./pkg/...
+test: build
+	@go test ./pkg/...
+	@go test -timeout 20m ./test
```

Comment:
`go test -v` output is a bit better, it shows which tests are running for each package

Reply:
I'm not a big fan of the -v option, as it tends to be overly verbose, and test failures end up buried in the verbose output, instead of nicely standing out when `-v` is not used.
I've changed the PR to add this regardless.